config/jobs/kubernetes/sig-auth/sig-auth-encryption-at-rest.yaml

```diff
@@ -40,10 +40,10 @@ presubmits:
         - ./test/e2e/testing-manifests/auth/encrypt/run-e2e.sh
         resources:
           limits:
-            cpu: 2
+            cpu: 4
             memory: 4Gi
```

Comment:
we should bump the memory to match, https://github.com/kubernetes/test-infra/issues/34139

on this CI cluster the machine is n1-highmem-8 so 8 gb / CPU, let's give this at least 12GB.

TODO: we should consider re-evaluating if highmem makes sense :^)

Reply:
sounds good! updated it.